easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-v1712-cleanup.patch

```diff
@@ -0,0 +1,615 @@
+# - Replace the OpenFOAM third-party libraries with EASYBUILD variables.
+# - Set install dir, compiler, etc using EASYBUILD
+#
+# <Mark.Olesen@esi-group.com>
```

Comment:
please don't change but extend the copyright on patches. This one is more or less the same as the one we already have?

Reply:
Sure - I don't actually care either way. The first patch was just a patch and someone requested that I add comment and copyright. So this time I added comments and copyright, but apparently not the right ones.
If it helps achieve what is required, I'm happy enough to work with a template if you what to provide one. 
However, I think that this whole patching business is a waste of time for OpenFOAM if we could instead just generate the config file contents directly (as per spack)

Reply:
@OleHolmNielsen I fully agree that the patching is a PITA, and that we should avoid it if possible. Let's follow up on this in #1357.

@wpoely86 Patch is fixed in 5f908e9